packages/daemon/tsconfig.json

```diff
@@ -10,6 +10,9 @@
     "outDir": "./dist",
     "types": ["node", "jest"]
   },
+  "references": [
+      { "path": "../../node_modules/@wallet-service/common" }
+  ],
```

Comment:
This makes tsconfig compile the common module tsconfig

Reply:
question(non-blocking): Should we add these instructions to a document on the repository itself?

There seem to be many of those small details that can break the production environment and delay future maintenance releases if left unchecked.

Reply:
I just found out that typescript implements its own JSON parser, which allows for comments in JSON

Added a comment in tsconfig.json in 536ee047812a2aeb8703120a5e3da64b016e0960

Reply:
Also created an issue so we document these details better

Reply:
https://github.com/HathorNetwork/hathor-wallet-service/issues/172

Reply:
Great! It will surely help with future maintenance